samples/snippets/create_lite_reservation_example.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to create a reservation using the
+Pub/Sub Lite API. For more information, see the root level README.md and
+the documentation at https://cloud.google.com/pubsub/lite/docs/reservations.
+"""
+
+import argparse
+
+
+def create_lite_reservation(
+    project_number, cloud_region, reservation_id, throughput_capacity,
+):
+    # [START pubsublite_create_reservation]
+    from google.api_core.exceptions import AlreadyExists
+    from google.cloud.pubsublite import AdminClient, Reservation
+    from google.cloud.pubsublite.types import CloudRegion, ReservationPath
+
+    # TODO(developer):
+    # project_number = 1122334455
```

Comment:
Does python support using project IDs? If so, should we prefer this over project numbers?

Reply:
To be consistent with the UI, Pub/Sub Lite uses project numbers. It's true of all existing Java/Go samples. 
![image](https://user-images.githubusercontent.com/9518298/153096357-9c821353-1856-4203-bf13-a1ac195fa729.png)